src/simulators/matrix_product_state/matrix_product_state_internal.cpp

```diff
@@ -364,10 +483,11 @@ void MPS::change_position(uint_t src, uint_t dst)
 cmatrix_t MPS::density_matrix(const reg_t &qubits) const
 {
   MPS temp_MPS;
-  uint_t front = 0, back = 0;
-  MPS_with_new_indices(qubits, temp_MPS, front, back);
-  MPS_Tensor psi = temp_MPS.state_vec(front, back);
+  uint_t new_first=0, new_last=0;
+  bool ordered = true;
```

Comment:
seems like you can remove `ordered` safetly

Reply:
done.